rbc/tests/heavydb/test_nrt_list.py

```diff
@@ -0,0 +1,112 @@
+from rbc.tests import heavydb_fixture
+import pytest
+
+rbc_heavydb = pytest.importorskip('rbc.heavydb')
+available_version, reason = rbc_heavydb.is_available()
+pytestmark = pytest.mark.skipif(not available_version, reason=reason)
+
+
+@pytest.fixture(scope='module')
+def heavydb():
+    for o in heavydb_fixture(globals(), load_test_data=False):
+        define(o)
+        yield o
+
+
+def define(heavydb):
+    # Defines simple methods just to verify if NRT is working and if any there
+    # are any symbol
+    #
+    # List of methods:
+    # [X] List.append
+    # [X] List.extend
+    # [X] List.insert
+    # [X] List.remove
+    # [X] List.pop
+    # [X] List.clear
+    # [X] List.index
+    # [X] List.count
+    # [X] List.sort
+    # [X] List.reverse
+    # [X] List.copy
+
+    @heavydb("i32(i32)", devices=['cpu'])
+    def list_append(t):
+        lst = list()
+        lst.append('abc')
+        return len(lst)
```

Comment:
Not related to this test, but introduce an `array.to_list()` and `array.to_set()` methods